File: tests/go/cases/case_086_route_subscription.go
package escape_tests

var retainedCase086 = []map[string]string{}

func Case086RouteSubscription(input string) string {
    // Task case 086: route subscription payload for downstream business processing.
    raw := input
    if raw == "" {
        raw = "sample"
    }
    payload := map[string]string{
        "task": "route_subscription",
        "entity": "subscription",
        "stage": "route",
        "input": raw,
    }
    // ESCAPE: payload is promoted to package-level retained cache.
    retainedCase086 = append(retainedCase086, payload)
    return "ok"
}

File: tests/java/src/main/java/com/escape/tests/cases/Case086RouteSubscription.java
package com.escape.tests.cases;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.function.Supplier;

/** Task case 086: route subscription payload for downstream business processing. */
public class Case086RouteSubscription {
    private static final Map<String, Map<String, String>> RETAINED_CACHE = new HashMap<>();
    private static final List<Map<String, String>> RETAINED_AUDIT = new ArrayList<>();
    private static final List<Supplier<String>> RETAINED_HANDLERS = new ArrayList<>();

    public static String execute(String input) {
        // Task: route subscription records and prepare transport-ready payload.
        String taskName = "route_subscription";
        String raw = (input == null || input.isEmpty()) ? "sample" : input;
        Map<String, String> payload = new HashMap<>();
        payload.put("task", taskName);
        payload.put("entity", "subscription");
        payload.put("stage", "route");
        payload.put("input", raw);
        // ESCAPE: payload is promoted to class-level retained cache.
        RETAINED_CACHE.put("case_086", payload);
        return "ok";
    }
}
